static/js/chartHandler.js

```diff
@@ -92,8 +92,7 @@ const format = input => {
     return 'r\u2080'
   }else if(input === 'ra'){
     return "r\u2090" 
-  }else if(input === 'force'){
-    return "Force"
   }
-  return input
+
+  return input.charAt(0).toUpperCase() + input.slice(1) 
```

Comment:
what is this for?

Reply:
This is so that the first Character is capitalized on the graph, because we are passing the values around as 'voltage' and not 'Voltage' ...

Reply:
ah, okay. thank you